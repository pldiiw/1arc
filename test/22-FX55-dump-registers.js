'use strict';

let engine = require('engine').initialize();
const dumpRegisters = require('instructions').dumpRegisters;

engine.data[0] = 54;
engine.data[1] = 128;
engine.data[2] = 4;
engine.data[3] = 230;
engine.data[4] = 18;
engine.I = 10;

const engine1 = dumpRegisters(engine, 4);

console.log(engine1.memory[10]);
console.log(engine1.memory[11]);
console.log(engine1.memory[12]);
console.log(engine1.memory[13]);
console.log(engine1.memory[14]);
console.log(engine1.I);
