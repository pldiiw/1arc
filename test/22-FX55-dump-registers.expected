54
128
4
230
18
15
